build_scripts/manageiq_tar_build.rb

```diff
@@ -59,7 +65,9 @@ def precompile_sti_loader
   end
 
   def build_service_ui
-    Dir.chdir(app_dir.join("manageiq-ui-service")) do
+    symlink_plugin_paths("manageiq-ui-service", ui_service_dir)
```

Comment:
This, in particular should probably be a rake task as part of manageiq-ui-service itself (but future PR)